diff -up telepathy-qt-0.9.8/cmake/modules/TelepathyDefaults.cmake.1~ telepathy-qt-0.9.8/cmake/modules/TelepathyDefaults.cmake
--- telepathy-qt-0.9.8/cmake/modules/TelepathyDefaults.cmake.1~	2020-03-24 01:51:20.550287635 +0100
+++ telepathy-qt-0.9.8/cmake/modules/TelepathyDefaults.cmake	2020-03-24 01:51:49.528191543 +0100
@@ -54,11 +54,7 @@ if(CMAKE_COMPILER_IS_GNUCXX)
         set(DEPRECATED_DECLARATIONS_FLAGS)
     endif ()
 
-    if(${TP_QT_NANO_VERSION} EQUAL 0)
-        set(NOT_RELEASE 0)
-    else()
-        set(NOT_RELEASE 1)
-    endif()
+    set(NOT_RELEASE 0)
 
     set(desired
         all
